src/applications/vaos/services/appointment/transformers.js

```diff
@@ -483,7 +483,8 @@ function getTypeOfCareById(id) {
   ];
 
   return allTypesOfCare.find(
-    care => care.idV2 === id || care.ccId === id || care.id === id,
+    ({ idV2 = '', ccId = '', id = '' }) =>
+      idV2 === inputId || ccId === inputId || id === inputId,
```

Comment:
This is a great solution to deconstruct and give default values to the id's to eliminate the false positive coming back